components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -93,6 +95,8 @@ public class JWTTokenIssuer extends OauthTokenIssuerImpl {
     private static final String INBOUND_AUTH2_TYPE = "oauth2";
     private Algorithm signatureAlgorithm = null;
 
+    private static final String AUTHORIZATION_CODE = "AuthorizationCode";
```

Comment:
Shall we define this static variable before defining signatureAlgorithm

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1984/commits/59ba3e10b73d02dcf0d68de230df5add71a2ab97

Removed this variable since there's no usage of that variable.